src/pages/VeSDL/LockedInfo.tsx

```diff
@@ -64,10 +70,16 @@ export default function LockedInfo(): JSX.Element {
           {aggSDLInfo ? formatBNToShortString(aggSDLInfo.totalVeSDL, 18) : "-"}
         </Typography>
       </Box>
-      {/* <Box flex={1}>
```

Comment:
You want to preserve this for the future?

Reply:
we're in the future